include/fmt/format-inl.h

```diff
@@ -538,7 +579,7 @@ class bigint {
     *this <<= exp;  // Multiply by pow(2, exp) by shifting.
   }
 
-  void square() {
+  FMT_CONSTEXPR20 void square() {
     int num_bigits = static_cast<int>(bigits_.size());
     int num_result_bigits = 2 * num_bigits;
     basic_memory_buffer<bigit, bigits_capacity> n(std::move(bigits_));
```

Comment:
With `FMT_CONSTEXPR20` defined as `constexpr`, line 588 no longer compiles on msvc because it calls a function (i.e. constructor) that's not allowed in a `constexpr` context.

Reply:
fixed, somehow I faced with `std::signbit`, but didn't face with `isfinite` and `isinf` 🤔